docs/openshift-on-azure-walkthrough.md

```diff
@@ -97,6 +97,8 @@ az group create --name $CLUSTER_NAME --location $LOCATION
 az openshift create --resource-group $CLUSTER_NAME --name $CLUSTER_NAME -l $LOCATION --fqdn $FQDN
 ```
 
+> List of azure regions that supports OpenShift on Azure : [Supported Regions](supported-regions.md).
```

Comment:
imo this shouldn't be greyed out. Rather better that it's in bold text so it can be easily spotted.